From 33c0ca4c1f6f3630a349156a6ad5d5e7faca567f Mon Sep 17 00:00:00 2001
From: Yuantian Tang <andy.tang@nxp.com>
Date: Tue, 27 Sep 2022 17:19:25 +0800
Subject: [PATCH 023/274] MLK-25894-1 dt-bindings: soc: add bindings for i.MX93
 SRC

Add bindings for i.MX93 System Reset Controller(SRC). SRC supports
resets and power gating for mixes.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Reviewed-by: Jacky Bai <ping.bai@nxp.com>
---
 include/dt-bindings/power/imx93-power.h | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 include/dt-bindings/power/imx93-power.h

diff --git a/include/dt-bindings/power/imx93-power.h b/include/dt-bindings/power/imx93-power.h
new file mode 100644
index 000000000..a4a029be1
--- /dev/null
+++ b/include/dt-bindings/power/imx93-power.h
@@ -0,0 +1,12 @@
++/* SPDX-License-Identifier: (GPL-2.0 OR MIT) */
++/*
++ *  Copyright 2022 NXP
++ */
++
++#ifndef __DT_BINDINGS_IMX93_POWER_H__
++#define __DT_BINDINGS_IMX93_POWER_H__
++
++#define IMX93_POWER_DOMAIN_MEDIAMIX		0
++
++#endif
+
-- 
2.25.1

